qiskit/transpiler/__init__.py

```diff
@@ -40,629 +41,787 @@
 Qiskit has four pre-built transpilation pipelines available here:
 :mod:`qiskit.transpiler.preset_passmanagers`.  Unless the reader is familiar with
 quantum circuit optimization methods and their usage, it is best to use one of
-these ready-made routines.
-
+these ready-made routines. By default the preset pass managers into 6 stages:
+
+#. ``init`` - any initial passes that are run before we start embedding the circuit to the backend
+#. ``layout`` - This stage runs layout and maps the virtual qubits in the circuit to the
+   physical qubits on a backend. See :ref:`layout_stage` for more details.
+#. ``routing`` - This stage runs after a layout has been run and will insert any
+   necessary gates to move the qubit states around until it can be run on
+   backend's compuling map. See :ref:`routing_stage` for more details.
+#. ``translation`` - Perform the basis gate translation, in other words translate the gates
+   in the circuit to the target backend's basis set. For more details on this stage you can refer to
+   :ref:`translation_stage`.
+#. ``optimization`` - The main optimization loop, this will typically run in a loop trying to
+   optimize the circuit until a condtion (such as fixed depth) is reached. See
+   :ref:`optimization_stage` for more details.
+#. ``scheduling`` - Any hardware aware scheduling passes. See :ref:`scheduling_stage` for more
+   details.
```

Comment:
The sentences here are a bit inconsistent whether they start with "This stage ..." or assume that's implied, so the tenses different between each.  You may also want to consider a definition list rather than a numbered list here, since that looks more like what you're laying out.  That would be
```rst
init
  Any initial passes that ...
  
layout
  This stage runs layout and ...
  
...
```

Reply:
I used a consistent style in: https://github.com/Qiskit/qiskit-terra/pull/9087/commits/a46e97de3261a7a394c0f63ff23f818f5936f49f although fwiw I just copied this from the `StagedPassManager` docs.